app/controllers/spree/admin/sale_prices_controller.rb

```diff
@@ -22,7 +22,7 @@ def create
 
       def destroy
         @sale_price = Spree::SalePrice.find(params[:id])
-        @sale_price.discard
+        @sale_price.send(SolidusSalePrices::SoftDelete.delete_method)
```

Comment:
Should probably use `public_send` throughout, just as a matter of safe practices, even if it has no impact to the overall function here.

Reply:
Removed send in favor of #discard. Please let me know if the new implementation works!